src/main/java/net/openhft/chronicle/network/cluster/handlers/UberHandler.java

```diff
@@ -299,4 +286,4 @@ public String toString() {
                 ", localIdentifier=" + localIdentifier +
                 '}';
     }
```

Comment:
Only reset heartbeat when we RECEIVE a message, not when we send one (we send regular heartbeats that will suppress a timeout if the other side stops sending heartbeats)